js/widgets/fixedToolbar.js

```diff
@@ -9,7 +9,9 @@
 
 //>>label: Toolbars: Fixed
 //>>group: Widgets
-//>>description: Behavior for "fixed" headers and footers - be sure to also include the item 'Browser specific workarounds for "fixed" headers and footers' when supporting Android 2.x or iOS 5
+//>>description: Behavior for "fixed" headers and footers - be sure to also include the item
+//>> 'Browser specific workarounds for "fixed" headers and footers' when supporting
+//>> Android 2.x or iOS 5
```

Comment:
Actually the workarounds file should be moved into backcompat since we dropped 2.3 in last meeting